tox.ini

```diff
@@ -28,6 +28,7 @@ envlist = py38,
 setenv =
     PIP_CONFIG_FILE={toxinidir}/pip.conf
     PYTHONASYNCIODEBUG=1
+    DISABLE_FALCON_CYTHON=Y
```

Comment:
Nitpick: it's better to group Falcon envvars under the `FALCON_` prefix, i.e., `FALCON_DISABLE_CYTHON`, `FALCON_ASGI_WRAP_NON_COROUTINES` etc.

Reply:
Sure